pkg/grpc/process_manager_test.go

```diff
@@ -171,6 +171,12 @@ func TestProcessManager_GetProcessExec(t *testing.T) {
 		Common: processapi.MsgCommon{
 			Ktime: 1234,
 		},
+		Creds: processapi.MsgGenericCredMinimal{
+			Uid:  0,
+			Gid:  0,
+			Euid: 0,
+			Egid: 0,
+		},
```

Comment:
0 is the default value,  processapi.MsgGenericCredMinimal{} has the same effect but is more concise.